Mathlib/RingTheory/MatrixAlgebra.lean

```diff
@@ -32,22 +100,22 @@ def toFunBilinear : A →ₗ[R] Matrix n n R →ₗ[R] Matrix n n A :=
 
 @[simp]
 theorem toFunBilinear_apply (a : A) (m : Matrix n n R) :
-    toFunBilinear R A n a m = a • m.map (algebraMap R A) :=
+    toFunBilinear n R A a m = a • m.map (algebraMap R A) :=
```

Comment:
Why is it desirable to change the argument order throughout the whole file? Is this following some existing convention?

Reply:
I've moved this to #21863 with an explanation.